File: p11crudNew/src/main/java/com/knowit/p11crud/Entity/Area.java
package com.knowit.p11crud.Entity;

import java.util.List;
import java.util.Set;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonManagedReference;

import jakarta.persistence.CascadeType;
import jakarta.persistence.Entity;
import jakarta.persistence.FetchType;
import jakarta.persistence.Id;
import jakarta.persistence.OneToMany;
import jakarta.persistence.Table;
import lombok.Data;

@Data
@Table(name="area")
@Entity
public class Area {
	
	  @Id
	  private Integer areaid;  // Area ID (Primary Key)
	  private String areaname;
	  private Integer pincode;
	  private Integer cityid;
	  
	  @JsonIgnoreProperties("areaid")
	  @OneToMany(mappedBy = "areaid",cascade = CascadeType.ALL)
	  private Set<User> users;
	   
	  @JsonIgnoreProperties("areaid")
	  @OneToMany(mappedBy = "areaid", cascade = CascadeType.ALL, fetch = FetchType.EAGER)
	   private List<Property> properties;

}

File: p11transaction/Models/Area.cs
using System;
using System.Collections.Generic;

namespace p11transaction.Models;

public partial class Area
{
    public int Areaid { get; set; }

    public string? Areaname { get; set; }

    public int Pincode { get; set; }

    public int? Cityid { get; set; }

    public virtual City? City { get; set; }

    public virtual ICollection<Property> Properties { get; set; } = new List<Property>();

    public virtual ICollection<User> Users { get; set; } = new List<User>();
}
